Source/Rivet.sln

```diff
@@ -1,11 +1,13 @@
 ﻿
-Microsoft Visual Studio Solution File, Format Version 13.00
-# Visual Studio 2013
-Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Rivet", "Rivet.csproj", "{E987DC29-C1B0-4C9A-B8A3-46823E636087}"
+Microsoft Visual Studio Solution File, Format Version 12.00

```

Comment:
Did we actually go down a version or were the old docs incorrect?

Reply:
I didn't touch this file. Visual Studio did. So, let's leave it as-is.